metrics/tikvclient.go

```diff
@@ -202,18 +193,16 @@ var (
 			Namespace: "tidb",
 			Subsystem: "tikvclient",
 			Name:      "batch_wait_duration",
-			// Min bucket is [0, 1ns).
-			Buckets: prometheus.ExponentialBuckets(1, 2, 30),
-			Help:    "batch wait duration",
+			Buckets:   prometheus.ExponentialBuckets(1, 2, 30), // 1ns ~ 1s
+			Help:      "batch wait duration",
 		})
 
 	TiKVBatchClientUnavailable = prometheus.NewHistogram(
```

Comment:
This metric doesn't be registered. Please help add it.

Reply:
added